legal/satzung.md

```diff
@@ -0,0 +1,122 @@
+# Satzung
+
+
+## Name, Sitz, Geschäftsjahr
+
+1.   Der Verein trägt den Namen *open energy modeling foundation* (oemof)
+2.   Sitz des Vereins ist $(SITZ).
+3.   Das Geschäftsjahr ist das Kalenderjahr.
+
+
+## Zweck und Ziele
+
+1. Der Verein verfolgt ausschließlich und unmittelbar gemeinnützige Zwecke im Sinne des Abschnitts „Steuerbegünstigte Zwecke“ der Abgabenordnung.
+2. Zweck des Vereins ist die Förderung von Wissenschaft und Forschung.
+    Dieser Zweck wird verwirklicht durch
+    1. Bereitstellung wissenschaftlicher Software unter OSI-anerkannten Open-Source-Lizenzen,
+    2. Unterstützung bei der Weiterentwicklung ebendieser Software, und
+    3. $(ZWECK)
+
+
+## Selbstlosigkeit
+
+1. Der Verein ist selbstlos tätig. Er verfolgt nicht in erster Linie eigenwirtschaftliche Zwecke.
+2. Die Mittel des Vereins dürfen nur für seine satzungsmäßigen Zwecke verwendet werden.
+    Die Mitglieder dürfen keine Gewinnanteile und in ihrer Eigenschaft als Mitglieder auch keine sonstigen Zuwendungen aus Mitteln des Vereins erhalten.
+3. Es darf keine Person durch Ausgaben, die dem Zweck des Vereins fremd sind, oder durch unverhältnismäßig hohe Vergünstigungen begünstigt werden.
+
+
+## Organe des Vereins
+
+1. Die Organe des Vereins sind
+    1. die Mitgliederversammlung und
+    2. der Vorstand.
+2. Der Vorstand besteht drei Mitgliedern,
+    1. der Sprecher*in,
+    2. der Schriftführer*in und
+    3. der Kassenwärt*in.
+4. Der Vorstand vertritt den Verein gerichtlich und außergerichtlich im Sinne des § 26 BGB,
+    wobei jedes Vorstandsmitglied für sich zur Vertetung des Vereins berechtigt ist.
+3. Der Vorstand gibt sich eine Geschäftsordnung, die von der Mitgliederversammlung bestätigt werden muss.
+
+
+## Treffen und Kommunikation
+
+1. Sämtliche Treffen des Vereins und seiner Organe können mittels Fernkommunikationsmittel unter Beachtung zumutbarer Identitätskontrollen durchgeführt werden.
+2. Benachrichtigungsfristen gelten als erfüllt, wenn die Benachrichtigung vor Verstreichen der Frist abgeschickt oder veröffentlicht wurde.
+3. Eine Benachrichtigung gilt als zugestellt, wenn sie an die letzte dem Verein bekannte Adresse verschickt oder auf der Webseite des Vereins veröffentlicht wurde wurde.
+4. Bei Zustimmung einer zu benachrichtigenden Person kann gegenüber dieser auf die Einhaltung von Benachrichtigungsfristen und -formalien verzichtet werden.
+
+
+## Mitgliedschaft
+
+1. Mitglied des Vereins kann jede volljährige natürliche Person werden.
+2. Die Beitrittserklärung erfolgt gegenüber dem Vorstand.
+2. Über Aufnahme entscheidet der Vorstand. Die Aufnahme erfolgt nach freiem Ermessen.
+    Bei Ablehnung ist er nicht verpflichtet, dem Antragsteller die Gründe hierfür mitzuteilen.
```

Comment:
This is to avoid tries to join by the decision of a court. The board would still be allowed (and might be encouraged) to asked for the opinion of other members.

Reply:
At the meeting of 22-03, there was consensus to keep it like it is, and to specify the details (like information of the members) in the Geschäftsordnung.